packages/babel-parser/test/fixtures/typescript/cast/as-const/input.js

```diff
@@ -0,0 +1,42 @@
+let v1 = 'abc' as const;
+let v2 = `abc` as const;
+let v3 = 10 as const;
+let v4 = -10 as const;
+let v5 = +10 as const;
+let v6 = 10 as const;
+let v7 = -10 as const;
+let v8 = true as const;
+let v9 = false as const;
+
+let a1 = [] as const;
+let a2 = [1, 2, 3] as const;
+let a3 = [10, 'hello', true] as const;
+let a4 = [...[1, 2, 3]] as const;
+let a5 = [1, 2, 3];
+let a6 = [...a5] as const;
+let a7 = [...a6];
+let a8 = ['abc', ...a7] as const;
+let a9 = [...a8];
+
+let o1 = { x: 10, y: 20 } as const;
+let o2 = { a: 1, 'b': 2, ['c']: 3, d() {}, ['e' + '']: 4 } as const;
+let o3 = { ...o1, ...o2 } as const;
+let o4 = { a: 1, b: 2 };
+let o5 = { ...o4 } as const;
+let o6 = { ...o5 };
+let o7 = { ...d } as const;
+let o8 = { ...o7 };
```

Comment:
What are these tests without `as const` needed for? I think that you copied them from https://github.com/Microsoft/TypeScript/blob/08fe06f5275bae91a2b2d7eb6387218b95ce3566/tests/baselines/reference/constAssertions.js, but since Babel doesn't type-check they aren't needed.

Reply:
Updated the test cases